src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -3800,9 +3800,12 @@ public void OutOfProcFileBasedP2PBuildSucceeds()
             }
         }
 
-        /// When a ProjectInstance based BuildRequestData is built out of proc, the node should
+        /// <summary>
+        /// When a <see cref="ProjectInstance"/> based <see cref="BuildRequestData"/> is built out of proc, the node should
         /// not reload it from disk but instead fully utilize the entire translate project instance state
-        /// to do the build
+        /// to do the build.
+        /// </summary>
+        /// <param name="shouldSerializeEntireState">Serializes the entire project instance state if true.</param>
```

Comment:
```suggestion
        /// <param name="shouldSerializeEntireState"><see langword="true"/> to serialize the entire project instance state; otherwise, <see langword="false"/>.</param>
```